java/add-point-scene-layer/README.md

```diff
@@ -1,8 +1,8 @@
-# Add point scene layer
+# Add a point scene layer
 
 View a point scene layer from a scene service.
 
-![Add point scene layer App](add-point-scene-layer.png)
+![Image of a point scene layer](AddAPointSceneLayer.png)
```

Comment:
let's not change the .png file names: if we do that then we'd have to update the png file name locally too and that adds extra overload.